package.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
Someone ran `npm init`. Remember `npm init` is only when we don't have a package.json file. If we already have one, we can just run `npm install` to get the dependencies.